### ISSUE TITLE
[bugfix] /root/.oh-my-zsh/oh-my-zsh.sh:187: unrecognized modifier on linux env

### ISSUE THREAD

**Issue by icefall7** [ohmyzsh/ohmyzsh#11598]
https://github.com/ohmyzsh/ohmyzsh/blame/1ad167dfac325a9f92e0693c70d0ab3f7c4c574b/oh-my-zsh.sh#L187

**Comment by isuftin** [comment]
Seeing the same issue.

```sh
zsh --version
zsh 5.7.1 (arm-unknown-linux-gnueabihf)

cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```

**Comment by icefall7** [comment]
i believe its the latest . i used the command sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"

and as i checked the line in oh-my-zsh.sh. these lines are updated in 13 mins ago.

**Comment by isuftin** [comment]
Note that this also removes the omz command from PATH

```sh
raspberrypi% omz
zsh: command not found: omz
```

So `omz update` is not possible :(

**Comment by mcornella** [comment]
Fixed in 9233ef75f28423d147a2f9f64a45a77b1886ee5d. You can manually upgrade with commands below:
```sh
cd "$ZSH"; git pull; exec -l zsh
```

**Comment by isuftin** [comment]
Works

```sh
raspberrypi% cd "$ZSH"; git pull; exec -l zsh
remote: Enumerating objects: 11, done.
remote: Counting objects: 100% (11/11), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 7 (delta 5), reused 7 (delta 5), pack-reused 0
Unpacking objects: 100% (7/7), done.
From https://github.com/ohmyzsh/ohmyzsh
   1ad167d..9233ef7  master     -> origin/master
Updating 1ad167d..9233ef7
Fast-forward
 oh-my-zsh.sh       | 6 +++---
 tools/changelog.sh | 3 +++
 2 files changed, 6 insertions(+), 3 deletions(-)
➜  .oh-my-zsh git:(master) omz update
Updating Oh My Zsh
         __                                     __
  ____  / /_     ____ ___  __  __   ____  _____/ /_
 / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \
/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / /
\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/
                        /____/

Oh My Zsh is already at the latest version.

To keep up with the latest news and updates, follow us on Twitter: https://twitter.com/ohmyzsh
Want to get involved in the community? Join our Discord: https://discord.gg/ohmyzsh
Get your Oh My Zsh swag at: https://shop.planetargon.com/collections/oh-my-zsh
```
